internal/core/cobra_utils_test.go

```diff
@@ -247,3 +271,32 @@ func Test_PositionalArg(t *testing.T) {
 		),
 	}))
 }
+
+func Test_MultiPositionalArg(t *testing.T) {
+	t.Run("set verb with one positional", core.Test(&core.TestConfig{
+		Commands: testGetCommands(),
+		Cmd:      "scw test multi-positional pos1 tag=tag1",
+		Check: core.TestCheckCombine(
+			core.TestCheckExitCode(0),
+			core.TestCheckGolden(),
+		),
+	}))
+
+	t.Run("set verb with multi positional", core.Test(&core.TestConfig{
+		Commands: testGetCommands(),
+		Cmd:      "scw test multi-positional pos1 pos2 pos3 tag=tag1",
+		Check: core.TestCheckCombine(
+			core.TestCheckExitCode(0),
```

Comment:
Can you check the result, it is expected that it contain list of arguments.
Last time I saw the error because of the golden but it could be missed in a larger PR.